tests/pipeline/test_alias.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+from zipline.testing.predicates import assert_equal
+from zipline.pipeline import Classifier, Factor, Filter
+from zipline.utils.numpy_utils import float64_dtype, int64_dtype
+
+from .base import BasePipelineTestCase
+
+
+class WithAlias(object):
```

Comment:
You can decorate with `nottest` to make nose not run the base class.  IMO, if you're defining tests methods in a class, you're no longer a fixture, you're a base class for other tests.

At the very least, I'd put an `init_class_fixtures` here asserting that any instantiated subclass of this is also a subclass of `BasePipelineTestCase`, since the tests here will fail in a non-obvious way if a consumer doesn't inherit from both.